source/jsonnet/england-wales/lib/placeholders.libsonnet

```diff
@@ -66,4 +66,62 @@
       },
     }],
   },
+  householdOrdinality: {
+    placeholder: 'household_ordinality',
+    transforms: [
+      {
+        transform: 'add',
+        arguments: {
+          lhs: { source: 'list', identifier: 'household' },
+          rhs: { value: 1 },
+        },
+      },
+      {
+        arguments: {
+          number_to_format: {
+            source: 'previous_transform',
+          },
+          determiner: {
+            value: 'a_or_an',
+          },
+        },
+        transform: 'format_ordinal',
+      },
+    ],
+  },
+  visitorOrdinality: {
+    placeholder: 'visitor_ordinality',
+    transforms: [
+      {
+        transform: 'add',
+        arguments: {
+          lhs: { source: 'list', identifier: 'visitors' },
+          rhs: { value: 1 },
+        },
+      },
+      {
+        arguments: {
+          number_to_format: {
+            source: 'previous_transform',
+          },
+          determiner: {
+            value: 'a_or_an',
+          },
+        },
+        transform: 'format_ordinal',
+      },
+    ],
+  },
+  householdCardinality: {
+    placeholder: 'household_cardinality',
+    transforms: [
+      {
+        transform: 'add',
+        arguments: {
+          lhs: { source: 'list', identifier: 'household' },
+          rhs: { value: 0 },
+        },
+      },
+    ],
+  },
```

Comment:
These are very similar and are essentially duplicates. These could be functions which allows us to pass in the list name i.e `cardinality(list_name)`. 

If we want to accept some duplication for the clear separation then I would be happy with that, but would be good to get thoughts on this. I'm not sure, if our style guide highlights this. We already have this pattern with `listIsEmpty` function.

@petechd @JamesGardiner @berroar @LJBabbage